charts/portworx/templates/portworx-controller.yaml

```diff
@@ -104,7 +104,9 @@ spec:
       - command:
         - kube-controller-manager
         - --leader-elect=true
+        {{- if (semverCompare ">=1.22" .Capabilities.KubeVersion.GitVersion) }}
```

Comment:
We need to remove the address field for k8s >= 1.22 and keep it as is for lower versions.
```
{{- if (semverCompare "<1.22" .Capabilities.KubeVersion.GitVersion) }}
```